### DISCUSSION TITLE
Support of Apple Silicon chips

### DISCUSSION THREAD

**Discussion by digicali** [coollabsio/coolify#1649] [Feature Requests]
More service and apps can run on Apple silicon servers with advantage of unified memory, neural engine and secure enclave. Example : macweb.com. Will be great to have a macOS version. Should not be difficult.

**Comment by peaklabs-dev** [comment]
@digicali We discussed this extensively in this issue: https://github.com/coollabsio/coolify/issues/2773. Can you please answer my question in the comments here: https://github.com/coollabsio/coolify/issues/2773#issuecomment-2311245860 as I am not sure if MacOs support makes sense due to the issues described (I'm thinking about adding it, but some insight will help).
